PubChatAndroid/app/src/main/AndroidManifest.xml

```diff
@@ -16,6 +16,12 @@
                 <category android:name="android.intent.category.LAUNCHER" />
             </intent-filter>
         </activity>
+        <activity android:name=".SignUpActivity">
+
+        </activity>
+        <activity android:name=".SignInActivity">
+
+        </activity>
```

Comment:
replace to : 
 <activity android:name=".SignUpActivity"/>
<activity android:name=".SignInActivity"/>